docs/editing_guidelines.md

```diff
@@ -178,3 +178,21 @@ The textual definition of a logically defined class should be a plain English eq
 ### Asserting relationships
 
 Please refer to the [relations guide](relations_guide.md) for detailled guidelines about which relations to use for most cases.
+
+## Design Pattern Usage with DOSDP
+
+
+The cell ontology is composed of a large number of terms, classifications and relationships, which continue to increase. As maintaining manually all these classifications and relationships would be an arduous task, a substantial portion of the maintenance is automated. This automation depends, to a significant extent, on the systematic application of design patterns.
+
```

Comment:
if you want, perhaps try:
The cell ontology contains a large number of terms, classifications, and relationships that are constantly expanding. Manually maintaining all these elements would be a daunting task, so a substantial portion of the maintenance is automated. This automation heavily relies on the systematic use of design patterns.